docs/03_server/11_integration/01_rest-endpoints/03_basics.mdx

```diff
@@ -79,232 +79,498 @@ Data Servers are the most complex of the REST Endpoints provided by the Genesis
 
 For all Data Server requests, a `SOURCE_REF` header is mandatory, as it is used to match requests to the correct client subscription. This in turn allows for multiple subscriptions to the same query for the same session (think multiple grids in the UI using the same query with different filters).
 
-There follows a list of Data Server messages mapped to HTTP endpoints.
+### Endpoint
+The naming approach for the dataserver endpoint follows the lowercase hyphen-separated web standards convention (i.e. ALL_TRADES becomes all-trades). There is no need to rename your dataserver resource, the endpoint will be exposed using the correct naming convention automatically.
 
-### DATA_LOGON
```

Comment:
I have divided the different headings into REST methods: POST, GET, PUT and DELETE. Given this is documentation about rest endpoints it feels more natural to define them that way.

Reply:
that makes sense.